doc/en/Packaging-status.md

```diff
@@ -4,18 +4,18 @@ RHVoice is available in the following Linux distributions:
 
 ## ArchLinux
 
-can be instaled from [AUR](https://aur.archlinux.org/)
+Can be installed from [AUR](https://aur.archlinux.org/):
 
-* [latest version](https://aur.archlinux.org/packages/rhvoice/)
-* [git(development) version](https://aur.archlinux.org/packages/rhvoice-git/)
+* [Latest version](https://aur.archlinux.org/packages/rhvoice/)
+* [Git version (development)](https://aur.archlinux.org/packages/rhvoice-git/)
```

Comment:
since there is a clarification in brackets, I think leaving it as it used to be would be correct.

```suggestion
* [Git(development) version](https://aur.archlinux.org/packages/rhvoice-git/)
```